plugins/janus_sip.c

```diff
@@ -760,45 +760,32 @@ int janus_sip_init(janus_callbacks *callback, const char *config_path) {
 		gboolean local_ip_set = FALSE;
```

Comment:
Good point: we had it because before `local_ip` was an array of characters, now that it's a pointer it's useless.